newIDE/app/src/MainFrame/Preferences/PreferencesDialog.js

```diff
@@ -80,10 +90,7 @@ const PreferencesDialog = ({ i18n, onClose }: Props) => {
         <Tabs
           value={currentTab}
           onChange={setCurrentTab}
-          options={[
-            { value: 'preferences', label: <Trans>Preferences</Trans> },
-            { value: 'shortcuts', label: <Trans>Keyboard Shortcuts</Trans> },
-          ]}
```

Comment:
"Fun thing" that is a bit more concise, you can do:

`...(electron ? [{ value: 'folders', label: <Trans>Folders</Trans> }] : [])`

to have "optional" things in an array and avoid deporting the array declaration outside the rendering code. Up to you!